---
title: "William Kumler"
knit: (function(inputFile, encoding) {
    rmarkdown::render(input = inputFile,
      encoding = encoding,
      output_format = "vitae::awesomecv",
      output_file = "william_kumler.pdf"
    )
  })
name: William
surname: Kumler
position: "Ph.D. Candidate"
address: "Seattle, WA"
email: "wkumler@uw.edu"
github: wkumler
urlcolor: blue
---

```{r setup, include=FALSE}
library(knitr)
knitr::opts_chunk$set(echo = FALSE)
word <- opts_knit$get("rmarkdown.pandoc.to")=="docx"
md <- opts_knit$get("rmarkdown.pandoc.to")=="gfm+tex_math_dollars-yaml_metadata_block"
latex <- opts_knit$get("rmarkdown.pandoc.to")=="latex"

eleg_space <- function(){
  word <- opts_knit$get("rmarkdown.pandoc.to")=="docx"
  md <- opts_knit$get("rmarkdown.pandoc.to")=="gfm+tex_math_dollars-yaml_metadata_block"
  # No idea why ifelse works when if;else doesn't but ok
  ifelse(md|word, ifelse(word, " \n<br>", " \n"), "\\hfill")
}
```

```{r, results='asis', include=FALSE}
alignment <- ifelse(word, "c", "l")
if(word|md){
  print(kable(data.frame(location="Seattle, WA"), col.names=NULL, align = alignment))
  metadata <- data.frame(
    email="wkumler@uw.edu",
    github="[github.com/wkumler](github.com/wkumler)",
    orcid="[ORCiD](https://orcid.org/0000-0002-5022-8009)"
  )
  print(kable(metadata, col.names = NULL, align=alignment))
}
```

```{r, results='asis'}
if(word|md){
  cat(paste(
    "Email: wkumler@uw.edu",
    "GitHub: [wkumler](https://github.com/wkumler)",
    "ORCiD: [0000-0002-5022-8009](https://orcid.org/0000-0002-5022-8009)",
    sep = "\n\n"
  ))
}
```


---

## Professional Summary

Ph.D. candidate in chemical oceanography focusing on marine microbial metabolomics under Dr. Anitra Ingalls. Expertise in automated and untargeted liquid chromatography mass spectrometry, with significant experience developing computational tools for mass spectrometry data analysis.

---

## Education

**Ph.D. in Chemical Oceanography**, University of Washington `r eleg_space()` _2019 - 2025_

  - Data Science Option

**M.Sc. in Oceanography**, University of Washington `r eleg_space()` _2019 - 2023_

**B.A. in Marine Science**, University of California, Berkeley `r eleg_space()` _2014 - 2018_

**B.S. in Molecular and Environmental Biology**, University of California, Berkeley `r eleg_space()` _2014 - 2018_

---

## Work Experience

**Ph.D. student**, Ingalls Lab, University of Washington, Seattle `r eleg_space()` _2019 - 2025_

- Published papers on marine microbial metabolomics with a focus on untargeted methodologies
- Developed existing and novel mass spectrometry tools for analysis and visualization
- Mentored graduate and undergraduate students in metabolomics
- Trained on Thermo Q Exactive HF with HILIC and reversed-phase chromatographies
- Will receive training on Thermo Orbitrap Astral in early 2025

**Lab Manager**, Edwards Lab, University of California, Berkeley `r eleg_space()` _2018 - 2019_

- Developed and maintained a lipidomics pipeline for data handling and statistical analysis
- Educated undergraduates in R programming and SLURM cluster usage
- Assisted with Thermo Fusion Lumos ID-X purchase and installation

---

## Publications

- **Kumler, W.**, Hazelton, B.J. & Ingalls, A.E. (2023). Picky with peakpicking: assessing chromatographic peak quality with simple metrics in metabolomics. _BMC Bioinformatics_, 24:404. [DOI:10.1186/s12859-023-05533-4](https://doi.org/10.1186/s12859-023-05533-4)
- **Kumler, W.** & Ingalls, A.E. (2022). Tidy data neatly resolves mass-spectrometry's ragged arrays. _R Journal_, 14(3), 193-202. [DOI:10.32614/RJ-2022-050](https://doi.org/10.32614/RJ-2022-050)
- Boysen, A.K., Durham, B.P., **Kumler, W.**, et al. (2022). Glycine betaine uptake and metabolism in marine microbial communities. _Environmental Microbiology_, 24(5), 2380-2403. [DOI:10.1111/1462-2920.16020](https://doi.org/10.1111/1462-2920.16020)
- **Kumler, W.E.**, Jorge, J., Kim, P.M., et al. (2020). Does formation of multicellular colonies by choanoflagellates affect their susceptibility to capture by passive protozoan predators? _Journal of Eukaryotic Microbiology_, 67(5), 555-565. [DOI:10.1111/jeu.12808](https://doi.org/10.1111/jeu.12808)

In review:

- **W. Kumler**, W. Qin, R.A. Lundeen, B. Barone, L.T. Carlson & A.E. Ingalls (in review with _Frontiers in Marine Science_). Metabolites reflect variability introduced by mesoscale eddies in the North Pacific Subtropical Gyre.
- E.A. Seelen, S.J. Gleich, **W. Kumler**, H.S. Anderson, X. Bian, K.M. Bjorkman, D.A. Caron, S.T. Dyhrman, S. Ferrón, Z.V. Finkel, S.T. Haley, Y. Hy, A.E. Ingalls, A.J. Irwin, D.M. Karl, K.P. Kong, D. Lowenstein, A. Salazar Estrada, E. Townsend, J.C. Tracey, K. Turk-Kubo, B.A.S. Van Mooy & S.G. John (in review with _Nature Communications_). A tale of two nutrients: how nitrogen and phosphorus differentially control production and stoichiometry.

In prep:

- **W. Kumler** & A.E. Ingalls. Quality assurance via bulk feature labeling for untargeted mass spectrometry data. Intended for _Analytical Chemistry_ in 2025. https://github.com/wkumler/squallms/blob/manuscript/paper.md

- **W. Kumler**, S. LaRue & A.E. Ingalls. Efficacy of SQL databases for storage, rapid search and retrieval of chromatographic mass spectrometry data. Intended for _Journal of Proteome Research_ in 2025. https://github.com/wkumler/mzsql.

- **W. Kumler**, R. Foreman, L.T. Carlson, D. Karl & A.E. Ingalls. Metabolic flux and fate of ^15^N depends on form in the North Pacific Subtropical Gyre. No journal yet selected.

---

## Technical Projects

- **RaMS**: R package for reading and tidying mass-spectrometry data. Available on [CRAN](https://cran.r-project.org/package=RaMS) and [GitHub](https://github.com/wkumler/RaMS).
- **pylgrams**: Python language gloss of the RaMS package. Available on [PyPI](https://pypi.org/project/pylgrams/) and [GitHub](https://github.com/wkumler/pylgrams).
- **squallms**: R package for interactive multi-file annotation of chromatographic feature quality. Available on [Bioconductor](https://bioconductor.org/packages/devel/bioc/html/squallms.html) and [GitHub](https://github.com/wkumler/squallms).

---

## Talks Given

- **Building a robust model of peak quality for untargeted mass-spectrometry** `r eleg_space()` _Apr. 2023_
    - Seminar given at UW's eScience Seminar (Recording available [here](https://www.youtube.com/watch?v=fNtp53o1wqQ&ab_channel=UWeScienceInstitute))
    - Seminar given at UW's Quantitative Science Seminar

- **Data Visualization** `r eleg_space()` _Oct. 2022_
    - Workshop led at the Graduate Climate Conference

- **Profiling R code and identifying performance bottlenecks** `r eleg_space()` _Nov. 2019_
    - Invited speaker at FSH 507 (Super-advanced R)

---

## Conference Proceedings

- **W. Kumler**, L.T. Carlson & A.E. Ingalls plus the PARAGON team (2024). Differences in 15N use by source during PARAGON. 
- J. Rainer, P. Louail, A. Vicini, R. Gine, J. Badia, M. Stravs, M. Garcia-Aloy, C. Huber, L. Salzer, J. Stanstrup, N. Shahaf, C. Panse, T. Naake, **W. Kumler**, P. Vangeenderhuysen, C. Brunius, H. Hecht, S. Neumann, M. Witting, S. Gibb, & L. Gatto (2024). An Open Software Development-based Ecosystem of R Packages for Metabolomics Data Analysis. 
- S. Garcia, F.X. Ferrer-González, J.S. Sacks, **W. Kumler**, L.T. Carlson, & A.E. Ingalls (2024). Characterizing the quality and quantity of metabolite production in phytoplankton cultures and the environment through endo- and exometabolomics. 
- **W. Kumler**, L.T. Carlson, & A.E. Ingalls plus the PARAGON team (2023). Metabolic fate of dissolved nitrogen in the NPSG. 
- **W. Kumler** (2022). Unmasking the POC/Ness Monster: Depth and Mesoscale Features Drive Variability in Particulate Metabolite Profiles of the MESOSCOPE transect. 
- **W. Kumler** & A.E. Ingalls (2021). RaMS: R-based Access to Mass-Spectrometry Data. 
- **W. Kumler**, H. Fredricks, J. Ossolinski, A. Allen, K. Thamatrakoln, K. Bidle, B. Van Mooy, & B. Edwards (2019). Sign of the times: the lipid signature of a collapsing phytoplankton bloom.

---


## References

  - Dr. Anitra E. Ingalls, Ph.D. Advisor. aingalls\@uw.edu, (206) 940-6755

  - Laura T. Carlson, Ingalls Lab Manager. truxal\@uw.edu, (412) 554-5093

